doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -38,7 +38,11 @@ ifdef::PAM[]
 
 It is now possible to set the priority of a task as an MVEL expression.
 
-== Process engine
+== {PROCESS_ENGINE_CAP}
+
+=== Support for exploded installation of Spring Boot business automation application
+
+It is now possible to get the support for exploded installation of Spring Boot business automation application. In case the application working directory is not the default one, you can set it with the `org.kie.maven.resolver.folder=/my-workspace/` variable. Note that for the Spring Boot image you must this variable into the  `JAVA_TOOL_OPTIONS: "-Dorg.kie.maven.resolver.folder=..."` environment variable.
```

Comment:
[Typo] Remove double space and add verb in reference to variable:

s/Note that for the Spring Boot image you must this variable into the  `JAVA_TOOL_OPTIONS:/Note that for the Spring Boot image you must add this variable to the `JAVA_TOOL_OPTIONS: